content/docs/concepts/multicluster-deployments/index.md

```diff
@@ -12,39 +12,94 @@ a single controllable composite application.
 
 For applications up to a certain size, all of the microservices comprising the
 application can be running on a single orchestration platform (e.g., Kubernetes cluster).
-However, for many reasons such as scale, redundancy, compliance, etc., most applications
-will eventually need to be distributed and have some of their services running elsewhere.
+Eventually, you may need to distribute most applications and run some or all of their services elsewhere due to one or more of the following reasons:
+
+- Scale
+- Redundancy
+- Workload isolation
+- Compliance
+- Integration with legacy services
 
 Istio supports many possible topologies for distributing the services of an
 application beyond a single cluster, for example:
 
-* Services within the mesh can use [service entries](/docs/concepts/traffic-management/#service-entries)
+- You can use [egress traffic control](/docs/tasks/traffic-management/egress/) to allow workloads inside the mesh
   to access standalone external services or to access services exposed by another loosely-coupled service mesh.
-* You can [expand the service mesh](/docs/setup/kubernetes/additional-setup/mesh-expansion/) to include
+- You can [expand the service mesh](/docs/setup/kubernetes/additional-setup/mesh-expansion/) to include
   services running on VMs or bare metal hosts.
-* You can combine the services from more than one cluster into a single composite service mesh.
+- You can combine the services from more than one cluster into a single composite service mesh.
+- You can combine autonomous service meshes in multiple clusters into a loosely coupled [federation of service meshes](#mesh-federation).
+
+## Single multicluster service mesh or mesh federation?
+
+As you plan your multicluster topology, you need to determine if you should combine the clusters into
+[a single multicluster mesh](#multicluster-service-mesh) or create [a federation of multiple meshes](#mesh-federation).
+To make that decision, ask yourself the following questions:
+
+1. Do you enforce **{{< gloss >}}uniform naming{{< /gloss >}}** in your clusters?
+
+1. Do you want your combined clusters to **expose** all their services to each other? Due to compliance or security
+   reasons, you might need {{< gloss >}}selective service exposure{{< /gloss >}}.
+
+1. Are you required to **control all the traffic that enters your cluster**?
+
+1. Will you be able to set **common CA root** for the Citadels running in your clusters?
```

Comment:
This restriction will go away soon (1.3 or 1.4), we'll support multiple roots in a single mesh with a single trust bundle for the entire mesh.

The more specific question is whether the clusters trust each others' roots. We can reword once that stuff is in place though, but maybe put something in that we're working on lifting this restriction?

Reply:
@smawson I propose to capture the current situation as it is on istio.io. Then to iterate on it, once new features/patterns are introduced/removed/simplified.

Reply:
Fair enough, would still like some forward-looking verbiage since we're working on it and this isn't inherent in "what is a mesh".